src/main/java/org/jabref/gui/maintable/MainTableHeaderRightClickMenu.java

```diff
@@ -0,0 +1,80 @@
+package org.jabref.gui.maintable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javafx.scene.control.ContextMenu;
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.RadioMenuItem;
+import javafx.scene.control.SeparatorMenuItem;
+import javafx.scene.control.TableColumn;
+import javafx.scene.layout.StackPane;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.LibraryTab;
+import org.jabref.gui.maintable.columns.MainTableColumn;
+import org.jabref.gui.preferences.PreferencesDialogView;
+import org.jabref.logic.l10n.Localization;
+
+public class MainTableHeaderRightClickMenu extends ContextMenu {
+    static MainTable mT = null;
```

Comment:
Why is this static? You could simply declare it as normal member variable.
I would pass the MainTable argument in the constructor and assign it there.

Reply:
Please do not abbreviate variable names.